### PR TITLE
login_ui and namespace selection for rosa hcp clusters

### PR DESCRIPTION
**Pull request by DanielOsypenko**
first adjustment for "rosa hcp" platform

### CODE REVIEW

**Comment by DanielOsypenko** [comment]
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/40994/

UI tests passed:
tests/cross_functional/ui/test_validation_ui.py::TestUserInterfaceValidation::test_storage_cluster_validation_ui | Passed
tests/cross_functional/ui/test_validation_ui.py::TestUserInterfaceValidation::test_dashboard_validation_ui | Passed
tests/cross_functional/ui/test_validation_ui.py::TestUserInterfaceValidation::test_odf_storagesystems_ui | Passed

**Review by ebondare — APPROVED** [review]

**Review by am-agrawa — APPROVED** [review]

File: ocs_ci/ocs/ui/validation_ui.py [hunk 1]
```diff
@@ -213,11 +213,12 @@ def odf_console_plugin_check(self):
         if default_projects_is_checked.get_attribute("data-checked-state") == "false":
             logger.info("Show default projects")
             self.do_click(self.validation_loc["show-default-projects"])
-        logger.info("Search for 'openshift-storage' project")
+        logger.info("Search for 'cluster_namespace' project")
```
Comment (dahorak) [inline ocs_ci/ocs/ui/validation_ui.py:216]: what about expanding the `cluster_namespace` here for the log message?
```suggestion
        logger.info("Search for '%s' project", config.ENV_DATA["cluster_namespace"])
```

File: ocs_ci/ocs/ui/validation_ui.py [hunk 2]
```diff
@@ -213,11 +213,12 @@ def odf_console_plugin_check(self):
...
         self.do_send_keys(
-            self.validation_loc["project-search-bar"], text="openshift-storage"
+            self.validation_loc["project-search-bar"],
+            text=config.ENV_DATA["cluster_namespace"],
         )
-        logger.info("Select 'openshift-storage' project")
+        logger.info("Select 'cluster_namespace' project")
```
Comment (dahorak) [inline ocs_ci/ocs/ui/validation_ui.py:221]: same as above:
```suggestion
        logger.info("Select '%s' project", config.ENV_DATA["cluster_namespace"])
```

**Review by dahorak — APPROVED** [review]
Just two small suggestions, otherwise LGTM.

**Review by dahorak — APPROVED** [review]

**Pull request merged**